cdk-infra/bulkExport/uploadGlueScriptsToS3.ts

```diff
@@ -56,7 +56,7 @@ exports.handler = async (event: any) => {
                     console.log(`uploading ${entry.filename} to ${process.env.GLUE_SCRIPTS_BUCKET}`);
                     return s3
                         .putObject({
-                            Bucket: process.env.GLUE_SCRIPTS_BUCKET,
+                            Bucket: process.env.GLUE_SCRIPTS_BUCKET!,
```

Comment:
Emmm why is the non-null assertion added here?

Reply:
Eslint was complaining that this value could possible be null, so I added the non-null assertion, since we check earlier to make sure that it's present before getting to this point.